specification/servicenetworking/resource-manager/Microsoft.ServiceNetworking/preview/2024-05-01-preview/TrafficController.json

```diff
@@ -0,0 +1,1900 @@
+        }
+      }
+    },
+    "SecurityPolicyUpdateProperties": {
+      "type": "object",
+      "description": "The updatable properties of the SecurityPolicy.",
+      "properties": {
+        "wafPolicy": {
+          "$ref": "#/definitions/WafPolicyUpdate",
+          "description": "Web Application Firewall Policy of the Traffic Controller Security Policy"
+        }
+      }
+    },
+    "TrafficController": {
+      "type": "object",
+      "description": "Concrete tracked resource types can be created by aliasing this type using a specific property type.",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/TrafficControllerProperties",
+          "description": "The resource-specific properties for this resource.",
+          "x-ms-client-flatten": true,
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ]
+    },
+    "TrafficControllerListResult": {
+      "type": "object",
+      "description": "The response of a TrafficController list operation.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "The TrafficController items on this page",
+          "items": {
+            "$ref": "#/definitions/TrafficController"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "format": "uri",
+          "description": "The link to the next page of items"
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "TrafficControllerProperties": {
+      "type": "object",
+      "description": "Traffic Controller Properties.",
+      "properties": {
+        "configurationEndpoints": {
+          "type": "array",
+          "description": "Configuration Endpoints.",
+          "items": {
+            "type": "string"
+          },
+          "readOnly": true
+        },
+        "frontends": {
+          "type": "array",
+          "description": "Frontends References List",
+          "items": {
+            "$ref": "#/definitions/ResourceId"
+          },
+          "readOnly": true
+        },
+        "associations": {
+          "type": "array",
+          "description": "Associations References List",
+          "items": {
+            "$ref": "#/definitions/ResourceId"
+          },
+          "readOnly": true
+        },
+        "securityPolicies": {
+          "type": "array",
+          "description": "Security Policies References List ",
+          "items": {
+            "$ref": "#/definitions/ResourceId"
+          },
+          "readOnly": true
+        },
+        "securityPolicyConfigurations": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SecurityPolicyConfigurations"
+          },
+          "readOnly": true,
```

Comment:
`securityPolicyConfigurations` shouldn't be read-only -- it needs to be writeable for the default security policy to get configured.